single_cell/annotation_import.R

```diff
@@ -0,0 +1,157 @@
+### This script provides a function for importing annotations from a csv file
+# Athough a relatively simple task, the goal is standardization and interfacing with annotation methodolgy of the Data Library
+#
+# (Documenting in roxygen syntax to be "library"-ready)
+#' Import annotations from a csv file into a Seurat or SCE object
+#' @param object A Seurat or SingleCellExperiment object, or a String representing the file path of an RDS file containing such an object
+#' @param annots_file String representing the file path of the annotations csv you wish to import from.
+#' See the details section for file structure requirements.
+#' @param annots_regex A String representing a regex that imported annotation columns will be expected to match.
```

Comment:
I think would be helpful to provide an example of a regex / e.g. the default, took me a second to understand why this parameter was here

Reply:
Added some more detail in the default example to be more explicit that the input defaults to "^annot", & gave an example of changing the regex where it would pull the "notes" column instead. =)